pandas/sparse/tests/test_frame.py

```diff
@@ -768,12 +768,20 @@ def _check(frame, orig):
         self._check_all(_check)
 
     def test_count(self):
-        result = self.frame.count()
         dense_result = self.frame.to_dense().count()
+
+        result = self.frame.count()
+        tm.assert_series_equal(result, dense_result)
+
+        result = self.frame.count(axis=None)
+        tm.assert_series_equal(result, dense_result)
+
+        result = self.frame.count(
+            axis=self.frame._stat_axis_number)
         tm.assert_series_equal(result, dense_result)
 
-        result = self.frame.count(1)
-        dense_result = self.frame.to_dense().count(1)
+        result = self.frame.count(axis=1)
```

Comment:
`np.cumsum` is just calling `sdf.cumsum(axis=None)`.  That's why the Exception is being thrown.